cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSIHDFSSinkTest.java

```diff
@@ -1196,5 +1212,352 @@ private Context createContext(String backendImpl, String backendMaxConns, String
+        String timestamp = "1461136795801";
+        String correlatorId = "123456789";
+        String transactionId = "123456789";
+        String originalService = "someService";
+        String originalServicePath = "somePath";
+        String mappedService = "newService";
+        String mappedServicePath = "newPath";
+        Map<String, String> headers = new HashMap<>();
+        headers.put(NGSIConstants.FLUME_HEADER_TIMESTAMP, timestamp);
+        headers.put(CommonConstants.HEADER_CORRELATOR_ID, correlatorId);
+        headers.put(NGSIConstants.FLUME_HEADER_TRANSACTION_ID, transactionId);
+        headers.put(CommonConstants.HEADER_FIWARE_SERVICE, originalService);
+        headers.put(CommonConstants.HEADER_FIWARE_SERVICE_PATH, originalServicePath);
+        headers.put(NGSIConstants.FLUME_HEADER_MAPPED_SERVICE, mappedService);
+        headers.put(NGSIConstants.FLUME_HEADER_MAPPED_SERVICE_PATH, mappedServicePath);
+        //NGSICKANSink.ColumnAggregator columnAggregator = ngsickanSink.new ColumnAggregator(false, false, false, false);
+        NGSIHDFSSink.CSVColumnAggregator columnAggregator = ngsihdfsSink.new CSVColumnAggregator();
+        NotifyContextRequest.ContextElement contextElement = createContextElementForNativeTypes();
+        NGSIEvent ngsiEvent = new NGSIEvent(headers, contextElement.toString().getBytes(), contextElement, null);
+        columnAggregator.initialize(ngsiEvent);
+        columnAggregator.aggregate(ngsiEvent);
+        System.out.println(getTestTraceHead("[NGSIHDFSSink.testNativeTypeColumnCsv]")
+                + "-  OK");
+        String correctAggregation = "2016-04-20T07:19:55.801Z,somePath,someId,someType,2,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeNumberxffffnumber/someId_someTypexffffsomeNumberxffffnumber.txt,true,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomneBooleanxffffBoolean/someId_someTypexffffsomneBooleanxffffBoolean.txt,2016-09-21T01:23:00.00Z,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeDatexffffDateTime/someId_someTypexffffsomeDatexffffDateTime.txt,\"{type: Point,coordinates: [-0.036177,39.986159]}\",hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeGeoJsonxffffgeo:json/someId_someTypexffffsomeGeoJsonxffffgeo:json.txt,{String: string},hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeJsonxffffjson/someId_someTypexffffsomeJsonxffffjson.txt,foo,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeStringxffffstring/someId_someTypexffffsomeStringxffffstring.txt,,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeString2xffffstring/someId_someTypexffffsomeString2xffffstring.txt";
+        if (columnAggregator.getAggregation().equals(correctAggregation)) {
+            assertTrue(true);
+        } else {
+            fail();
+            System.out.println(getTestTraceHead("[NGSIHDFSSink.testNativeTypeColumnCsv]")
+                    + "-  FAIL");
+        }
+
+    }//testNativeTypeColumnCsv
```

Comment:
```suggestion
    } // testNativeTypeColumnCsv
```

To keep same style than existing ones

Reply:
Fixed in [9e61357](https://github.com/telefonicaid/fiware-cygnus/pull/1799/commits/9e61357a49559bffebc01a71389c87be49ebd72c)